roles/edpm_network_config/tasks/os_net_config.yml

```diff
@@ -16,7 +16,6 @@
 #
 # Apply network configuration with os-net-config.
 #
```

Comment:
i'd prefer to keep a newline between header and taskfile content.

Reply:
ack unintentional restoring